GoogleForCreators/web-stories-wp
# Floating menu: Keyboard Navigation to Enter and Exit

## Context
Part of accessibility work on canvas and floating menu


## Summary
Adds `ctrl+alt+p` to enter into the floating menu when an element is focused and selected, `esc` to return focus back to the original element in the frames layer.


## Relevant Technical Choices
NA


## To-do
NA


## User-facing changes
User can now navigate into the floating menu with keyboard by pressing `ctrl+alt+p` while an element is selected. They can navigate out of the floating menu and back to the element by pressing `esc`.


## Testing Instructions






This PR can be tested by following these steps:

1. with an element selected & focused, press `ctrl+alt+p` and see that focus goes inside the floating menu
2. now press `esc` and see that focus leaves the floating menu


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10872
